apps/www/components/examples/breadcrumb/with-routing-library.tsx

```diff
@@ -0,0 +1,32 @@
+import Link from "next/link"
+
+import { siteConfig } from "@/config/site"
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+} from "@/components/ui/breadcrumb"
+import { Icons } from "@/components/icons"
+
+export function BreadcrumbWithRoutingLibrary() {
+  return (
+    <Breadcrumb>
+      <BreadcrumbItem>
+        <BreadcrumbLink as={Link} className="flex items-center gap-2" href="/">
```

Comment:
Would this also work with [react-router Links](https://reactrouter.com/en/main/components/link)?
If yes would it be possible for you to add another example file demonstrating this?